### DISCUSSION TITLE
No description provided for Docker image

### DISCUSSION THREAD

**Discussion by ammnt** [community/community#49090] [Packages] [answered]
### Select Topic Area

Question

### Body

Hello everyone!

I created the Docker image via GitHub Actions and added LABEL into Dockerfile:
https://github.com/ammnt/nginx/blob/http3/Dockerfile

but I still have "No description provided" in Package section:
https://github.com/ammnt/nginx/pkgs/container/nginx/74729755?tag=http3

Docker inspect showed everything fine:
![image](https://user-images.githubusercontent.com/45385632/222758035-9354a05e-0753-4a72-b7e1-7e2e49af8c5e.png)

Please help me with this issue🤔What do I need to do?

Cheers!🙋🏻‍♂️

**Comment by dekoeky** [comment]
Hello @ammnt,

Looking at [your container image](https://github.com/ammnt/nginx/pkgs/container/nginx/116040280?tag=http3) it seems you solved your issue. 
Could you tell me how you did it?

Thanks! :)

**Reply by ammnt** [reply, marked as answer]
Hi @dekoeky, as far as I remember, I first added a description of the project to the "About" section of my repository, then contacted to GH support and they recommended me to update the "Actions" version and add this directive:
https://github.com/ammnt/nginx/blob/http3/.github/workflows/build.yml#L58

Good luck🙂
